src/App.jsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
+import { Sum } from './components/Sum/Sum';
 import './App.scss';
 
-const App = () => (
-  <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
-  </>
+export const App = () => (
+  <div className="output">
+    <>
+      <Sum
+        a={2}
+        b={3}
+      />
+      <Sum
+        a={-5}
+        b={5}
+      />
+      <Sum
+        a={10}
+      />
+      <Sum
+        b={5}
+      />
+      <Sum />
+    </>
```

Comment:
Some advice about code style:
1. You already have enclosing `div`. No need to add react fragment here.
2. Same rules as for HTML attributes apply for React. If we have 2 or less - all can be put in one line. If there are three or more - use new lines.
```suggestion
    <Sum a={2} b={3} />
    <Sum a={-5} b={5} />
    <Sum a={10} />
    <Sum b={5} />
    <Sum />
```